fjord/feedback/tests/test_api.py

```diff
@@ -1,3 +1,5 @@
+# This Python file uses the following encoding: utf-8
+
```

Comment:
Had to set the encoding of the file to avoid an exception due to the presence of Unicode characters in this file